walnut/cmd/check.go

```diff
@@ -37,25 +38,18 @@ import (
 var checkCmd = &cobra.Command{
 	Use:   "check",
 	Short: "check the file passed against a specified schema.",
```

Comment:
`validate the file against a specified schema`

Also please make sure that all your `Short` descriptions don't end with a period. IIRC the Cobra docs have some advice on wording (though now I can't find it).